README.md

```diff
@@ -222,10 +213,8 @@ using the following,
     #> 
     #> To cite covidregionaldata in publications use:
     #> 
-    #>   Sam Abbott, Katharine Sherratt, Joe Palmer, Richard Martin-Nielsen,
-    #>   Jonnie Bevan, Hamish Gibbs, and Sebastian Funk (2020).
-    #>   covidregionaldata: Subnational Data for the COVID-19 Outbreak, DOI:
-    #>   10.5281/zenodo.3957539
+    #>   Sam Abbott, Katharine Sherratt, Joe Palmer, Richard Martin-Nielsen, Jonnie Bevan, Hamish Gibbs, and Sebastian Funk (2020). covidregionaldata: Subnational Data for the COVID-19 Outbreak,
```

Comment:
Does the JOSS tag come with DOI if yes should switch over the citation.

Reply:
full tag is `Markdown: [![status](https://joss.theoj.org/papers/dd6f7acdae3b7136a3ac373ce9a0655c/status.svg)](https://joss.theoj.org/papers/dd6f7acdae3b7136a3ac373ce9a0655c)`

Came from [here](https://github.com/openjournals/joss-reviews/issues/3278)